src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final int DATE_INDEX = 0;
+    static final int NAME_INDEX = 1;
+    static final int HOUR_INDEX = 2;
+    static final int SUM_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+        LocalDate dateStart = LocalDate.parse(dateFrom, dateTimeFormatter);
+        LocalDate dateEnd = LocalDate.parse(dateTo, dateTimeFormatter);
+        int sumSalary;
+        String[] salaryData;
+        LocalDate dateOfSalary;
```

Comment:
No need to declare variables in advance

Reply:
in Java, you should declare and initialize variables at the same time. Don't just declare them. Because you can have an initialization exception if you try to do anything with it